quack_app/lib/screens/friends/friends_screen.dart

```diff
@@ -26,27 +25,48 @@ class _FriendsScreenState extends State<FriendsScreen> {
       body: Column(
         children: [
           const SizedBox(height: 15),
-          Align(
-            alignment: Alignment.topCenter,
-            child: ElevatedButton(
-              onPressed: () async {
-                await Navigator.push(context,
-                    MaterialPageRoute(builder: (context) {
-                  return const SetLocationScreen();
-                }));
-                setState(() {});
-              },
-              child: AutoSizeText(
-                  "Share Location (${User().userData.getSeatingLocString()})"),
-            ),
```

Comment:
Weird that GitHub thinks you deleted all this when you just nested it in something else.